src/class-endpoints.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * This file registers REST API Endpoints.
+ *
+ * @package wp-conditional-blocks
+ */
+
+namespace Alley\WP\WP_Conditional_Blocks;
+
+use Alley\WP\Types\Feature;
+
+/**
+ * Class Endpoints
+ *
+ * This class implements the Feature interface and registers REST API Endpoints.
+ */
+class Endpoints implements Feature {
+	/**
+	 * Boot the feature.
+	 *
+	 * @return void
+	 */
+	public function boot(): void {
+		new Endpoint_Get_Conditions();
+	}
+}
```

Comment:
I'll try to provide some more context on the feature thing.

Since this plugin was created, we've added into create-wordpress-plugin.

See the readme at https://github.com/alleyinteractive/create-wordpress-plugin/blob/develop/src/features/README.md

The feature class should contain everything in one file for that feature (except for dependencies, etc), so that it can be easily turned on and off and is portable.

Features are loaded in the main plugin file like this:
See https://github.com/alleyinteractive/create-wordpress-plugin/blob/develop/plugin.php#L65-L73

There's also a Feature_Manager class in create-wordpress-plugin that should make things easier.

For some real world examples, check out this feature https://github.com/alleyinteractive/mediaco/blob/production/plugins/mediaco/src/features/class-subheadline.php
and then it's added to an array at https://github.com/alleyinteractive/mediaco/blob/production/plugins/mediaco/mediaco.php#L168
and the array of features is added via the Feature_Manager here: https://github.com/alleyinteractive/mediaco/blob/production/plugins/mediaco/mediaco.php#L226-L227

I'm happy to swarm with you on this, or we can split all this off into a separate issue and refactor it all there.

Reply:
Let's swarm on this Greg!